chart/compass/values.yaml

```diff
@@ -139,7 +139,7 @@ global:
       name: compass-pairing-adapter
     director:
       dir:
-      version: "PR-2868"
```

Comment:
Update the versions of all other components, because you've changed the go.sum/mod files.